src/_gettsim/sozialversicherung/arbeitslosen/arbeitslosengeld.py

```diff
@@ -124,3 +135,45 @@ def einkommen_vorjahr_proxy_m(
     )
     out = max_wage - prox_ssc - prox_tax / 12 - prox_soli / 12
     return max(out, 0.0)
+
+
+@param_function(start_date="1997-03-24")
+def anspruchsdauer_nach_alter(
+    raw_anspruchsdauer_nach_alter: dict[int, int],
+) -> ConsecutiveInt1dLookupTableParamValue:
+    """Amount of potential months of unemployment benefit claims by age."""
+    max_age = 120
```

Comment:
We put these numbers into the yaml files for other parameters, right? Shouldn't we do that here, too?